subsys/net/lib/nrf_cloud/include/nrf_cloud_mem.h

```diff
@@ -12,31 +12,31 @@ extern "C" {
 #endif
 
 #include <zephyr/kernel.h>
+#include <net/nrf_cloud_os.h>
 
-/**@brief Method to allocate zero-initialized memory of size COUNT*SIZE, for
+/**@brief Method to allocate zero-initialized memory of size count*size
```

Comment:
Usually, the function descriptions need to have an imperative form (do something). See https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#functions.

Reply:
Ok, changed